openfisca_france/model/prestations/mobilite_jeunes.py

```diff
@@ -0,0 +1,68 @@
+from openfisca_france.model.base import Individu, Variable, MONTH
+from openfisca_france.model.prestations.education import TypesClasse
+
+
+class aide_mobilite_parcoursup_sortie_academie(Variable):
+    value_type = bool
+    label = "Indicatrice d'une sortie de l'académie d'études au lycée dans les voeux ParcourSup"
```

Comment:
Évitons de mentionner le nom d'un logiciel spécifique dans une documentation législative.

```suggestion
    label = "Mention d'un vœu de changement d'académie entre le lycée et les études supérieures"
```